lib/deck.rb

```diff
@@ -10,22 +10,21 @@ def add_card(card)
   end
 
   def rank_of_card_at(index)
-    @cards[index].rank
+    cards[index].rank
   end
 
   def high_ranking_cards
-    highest = @cards.find_all do |card|
+    cards.find_all do |card|
       card.rank >= 11
     end
-    highest
   end
 
   def percent_high_ranking
     (percent_math * 100).round(2)
   end
 
   def percent_math
-    high_ranking_cards.length / @cards.length.to_f
```

Comment:
the `@` sign usually isn't needed, as long as you've got the `attr_reader`. If you're _changing_ the value, you'll need the `@` sign